.php_cs

```diff
@@ -1,18 +1,25 @@
 <?php
 
-$finder = Symfony\CS\Finder\DefaultFinder::create()
+$finder = PhpCsFixer\Finder::create()
     ->exclude('migrations')
     ->in(__DIR__);
 
-return Symfony\CS\Config\Config::create()
+$config = PhpCsFixer\Config::create()
     ->setUsingCache(true)
-    ->level(Symfony\CS\FixerInterface::PSR2_LEVEL)
-    ->fixers([
-        '-psr0',
-        'multiline_array_trailing_comma',
-        'ordered_use',
-        'short_array_syntax',
-        'unused_use',
+    ->setRules([
+        '@PSR2' => true,
+        'array_syntax' => [
```

Comment:
Fixers have become rules, some of which can be configured, others can just be enabled or disabled,